src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -623,10 +635,6 @@ bool fx_muxer_t::resolve_sdk_dotnet_path(const pal::string_t& own_dir, pal::stri
             }
         }
     }
```

Comment:
The logging is just moved as part of the tiny refactoring to share code. The traditional caller queries cwd, logs same message as before on failure, and then passes it down.